open-in-vscode/src/open-in-vscode.c

```diff
@@ -90,6 +90,52 @@ char *addToCommand(char *command, const char *path)
   return newCommand;
 }
 
+void handleMimeType(const char *mimeType, char **command, GList *files, size_t *nrOfMimeType) {
```

Comment:
`nrOfMimeType` should probably be something like `nrSelectedFiles`

Reply:
Also the generell input parameters should come before output parameters. :)

Reply:
> `nrOfMimeType` should probably be something like `nrSelectedFiles`

I don't think so, cause the mimeType can also be something like a directory (inode).

Reply:
> > `nrOfMimeType` should probably be something like `nrSelectedFiles`
> 
> I don't think so, cause the mimeType can also be something like a directory (inode).

Then probably something like nrSelectedEntries, because we are not selecting mime types but _by_ mime type.
But since this is mostly semantics, lets mark this as resolved.